tests-perf/locust/locust-notifications.py

```diff
@@ -1,24 +1,102 @@
+""" locust-notifications.py
+    isort:skip_file
+"""
+# flake8: noqa
+
 import os
+import sys
+from datetime import datetime
+from dataclasses import make_dataclass
+
+sys.path.append(os.path.abspath(os.path.join("..", "tests_smoke")))
 
 from dotenv import load_dotenv
 from locust import HttpUser, constant_pacing, task
+from tests_smoke.smoke.common import job_line, rows_to_csv  # type: ignore
 
 load_dotenv()
-AUTH_HEADER = os.getenv("TEST_AUTH_HEADER")
+NotifyApiUserTemplateGroup = make_dataclass('NotifyApiUserTemplateGroup', [
+    'bulk_email_id',
+    'email_id',
+    'email_with_attachment_id',
+    'email_with_link_id',
+    'sms_id',
+])
 
 
 class NotifyApiUser(HttpUser):
 
     wait_time = constant_pacing(60)
-    host = "https://api.staging.notification.cdssandbox.xyz"
+    host = os.getenv("LOAD_TEST_DOMAIN", "https://api.staging.notification.cdssandbox.xyz")
+
+    def __init__(self, *args, **kwargs):
+        super(NotifyApiUser, self).__init__(*args, **kwargs)
+
+        self.headers = {"Authorization": os.getenv("TEST_AUTH_HEADER")}
+        self.email = os.getenv("LOAD_TEST_EMAIL", "success@simulator.amazonses.com")
+        self.phone_number = os.getenv("LOAD_TEST_PHONE_NUMBER", "12897682684")
```

Comment:
That phone number might incur a cost but we can use one listed [on that page](https://documentation.notification.canada.ca/en/testing.html) which will skip the sending part.

Reply:
This has been changed